src/worldofzuul/Room.java

```diff
@@ -74,4 +90,30 @@ private String getExitString() {
     public Room getExit(String direction) {
         return exits.get(direction);
     }
+    
+    /**
+     * This method is a getter method for the items
+     * @return returns the item
+     */
+    public Item getItems() {
+       return item; 
+    }
+    
+    /**
+     * This method adds items to our ArrayList
+     * @param items
+     * @return returns an item into our ArrayList
+     */
+    public boolean addItems(ArrayList items) {
+       return items.add(item);
+    }
```

Comment:
If we change the items type from `ArrayList` to `Array`, we can more easily set the items with the array shorthand like below.
```java
room.addItems({
     new Ketchup(),
     new Computer()
})
```

Reply:
fixed in: 718f5ca2b3d2a167093b40a4fc2b4826fc510239